README.md

```diff
@@ -44,9 +44,15 @@ Facebook has adopted a Code of Conduct that we expect project participants to ad
 
 Read our [contributing guide](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to Docusaurus.
 
-### Contribute online with a single click
+### Online one-click setup for contributing
```

Comment:
I would prefer not to make new changes to this file,  since this duplicates what is written in the `CONTRIBUTING` file, and is not particularly useful information for `README` file, IMO.

Reply:
Thanks! I have made this change.